app/modules/users/resources.py

```diff
@@ -391,6 +395,26 @@ def get(self, user):
             return send_file(profile_path, attachment_filename='profile_image.jpg')
 
 
+@api.route('/reset_password_email')
+class UserResetPasswordEmail(Resource):
+    def post(self):
+        try:
+            email = json.loads(request.data)['email']
+        except (json.decoder.JSONDecodeError, TypeError, KeyError):
+            abort(
+                400, 'JSON body needs to be in this format {"email": "user@example.org"}'
+            )
+        user = User.find(email=email)
+        if not user:
+            abort(400, 'User with email address {repr(email)} not found')
```

Comment:
there are [decades of flamewars](https://ux.stackexchange.com/questions/13516/how-to-tell-a-user-their-login-credentials-are-incorrect) about this topic, so i hesitate to bring it up.... but....  i had always learned it is better to not leak information around logins/email/passwords in your responses to failed matches like this.  e.g. better to simply report something like _"if there is a user on our system matching this, an email will be sent"_ to any email address passed.
thats just me though.  :smile:   i am very content to bow to the whims of the team on this one.

Reply:
We can add logs in houston so if we need it for development, we can see it but our API won't expose it